ChiTech/ChiMesh/UnpartitionedMesh/lua/create.cc

```diff
@@ -39,14 +41,18 @@ int chiUnpartitionedMeshFromVTU(lua_State* L)
 {
   const char func_name[] = "chiUnpartitionedMeshFromVTU";
   int num_args = lua_gettop(L);
-  if (num_args != 1)
+  if (num_args < 1)
     LuaPostArgAmountError(func_name,1,num_args);
 
```

Comment:
Please add two lines here. 

```
LuaCheckNilValue(func_name,L,1);
if (num_args >=2) LuaCheckNilValue(func_name,L,2);
```

Reply:
These are checks that are actually needed across-the-board. Let's try and add them wherever we can.

Reply:
You must have anticipated my response. Included the verification in all four methods defined in the file in question.